Notebooks/QC/index.ipynb

```diff
@@ -144,20 +133,16 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "java -jar $PICARD -h"
+    "picard FastqToSam -h"
```

Comment:
just `picard -h`
There will be other `picard` commands in the tutorial that need to be changed.